clients.md

```diff
@@ -23,7 +23,7 @@ The filtering can be done either using the OPTIMADE filter language (see the spe
 
 ## optimade.science
 
-An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Tilde MI](https://tilde.pro){:target="_blank"} utilizing the [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available as a single universal `index.html` file. Can be run from server, USB-stick, etc.
-It allows for searching through *multiple* OPTIMADE databases using the OPTIMADE filter language (see the specification for more information).
+An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Tilde MI](https://tilde.pro){:target="_blank"} utilizing the [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available as a single universal `index.html` file. Can be run from the server, USB-stick, etc.
+It allows for searching through *all* the known OPTIMADE databases *simultaneously* using the OPTIMADE filter language (see the specification for more information).
```

Comment:
```suggestion
It allows for searching through *all* the known OPTIMADE databases from the [OPTIMADE providers list](https://providers.optimade.org) *simultaneously* using the OPTIMADE filter language (see the specification for more information).
```

Reply:
And even then it's a bit untrue, since there are some listed in the providers list which are not available on optimade.science.

Reply:
OK, we can write _all the official_, no problem.

Concerning the possible mismatches: this is due to caching. The *optimade.science* isn't updated as frequently as the source on GitHub. Now the cache is synced with each new release of the [isomorphic client](https://github.com/tilde-lab/optimade-client). Although suboptimal, this was an easiest option, subject to change, of course. In fact, retrieving the providers list must be very quick, and we cannot do that on-the-fly in the browser: the recursive lookup takes about one minute. No one will wait for a minute during the page load. So here's why 😊